depthai_sdk/docs/source/blob_nnet_manager.rst

```diff
@@ -0,0 +1,76 @@
+======================
+Blob and Nnet managers
+======================
+
+Blob manager
+------------
+
+``Blob manager`` is a class that is made to help you with downloading neural networks as MyriadX blobs.
+
+Getting started
+^^^^^^^^^^^^^^^
+
+``Blob manager`` is very easy and straight forward to use. We declare it and pass which project we want to use as it's argument.
+
+.. literalinclude:: ./examples/code_fractions/blob.py
+   :language: python
+   :linenos:
+
+After that, the ``blob`` is stored in our variable and we can then pass it to our ``Nnet manager``, as we will se bellow.
+Of course the ``face-detection-retail-0004`` is not the only project that we can use. All other projects that are supported are:
+
+.. include::  blob_projects.rst
+
+Nnet manager
+------------
+
+``Blob manager`` is a class that is made to help you with setting up neural networks.
```

Comment:
typo, nnet manager. This description should be extended, as NNetManager is responsible for all NN-related efforts, including creating a correct pipeline as well as drawing nn results on frames